src/workers/userActivity.worker/inactivityTracker/queries/queries.js

```diff
@@ -1,5 +1,7 @@
 const getLastAppOpenedTimestamp = `query getUsers{
-  user_actions{
+  user_actions(order_by: {
+    last_app_opened_timestamp: asc_nulls_last
+  }, limit:1){
```

Comment:
Please remove the limit & let's implement paginated retrieval, as the max records returned by hasura is 1000